src/elements/ChrPlasmaLens.H

```diff
@@ -152,21 +152,43 @@ namespace impactx::elements
             amrex::ParticleReal pyout = py;
             amrex::ParticleReal const ptout = pt;
 
-            // paceholder variables
+            // placeholder variables
             amrex::ParticleReal q1 = x;
             amrex::ParticleReal q2 = y;
             amrex::ParticleReal p1 = px;
             amrex::ParticleReal p2 = py;
 
-            auto const [sin_ods, cos_ods] = amrex::Math::sincos(omega*m_slice_ds);
-
-            // advance transverse position and momentum (focusing)
-            xout = cos_ods * x + sin_ods / (omega * delta1) * px;
-            pxout = -omega * delta1 * sin_ods * x + cos_ods * px;
-
-            yout = cos_ods * y + sin_ods / (omega * delta1) * py;
-            pyout = -omega * delta1 * sin_ods * y + cos_ods * py;
-
+            if (m_g > 0)
```

Comment:
Just to use the right literal type:
```suggestion
            if (m_g > 0_prt)
```

Reply:
Done on my changes.

Should this really be applied to every single litteral in the code? E.g. also here:
https://github.com/BLAST-ImpactX/impactx/pull/1030/files#diff-3858396255f74cba0b31e856f293d63c9ee64f702310a4466d58a0c1c8cfe451R63 , in ConstF constructor argument list, which includes the code
```
amrex::ParticleReal dx = 0,
amrex::ParticleReal dy = 0,
amrex::ParticleReal rotation_degree = 0,
amrex::ParticleReal aperture_x = 0,
amrex::ParticleReal aperture_y = 0,
```

Reply:
For immediate assignments it does not matter that much, especially for zeros.

It is more problematic when passed into math functions and math terms, which causes doubly up and down-casting when we compile for single precision.

I just checked, and comparison to zero is actually no problem as well: https://godbolt.org/z/qYn8rfK8v

Sorry for the unnecessary comment!